docs/manuals/gui/viselements/charts/heatmap.md

```diff
@@ -1,3 +1,441 @@
-## Heatmaps
+# Heatmaps
 
-TODO
+A heatmap depicts values for a main variable of interest across two axes as a grid of colored
+rectangles. The axes are divided into ranges like a bar chart or histogram, and each cell’s
+color hints at the value of the main variable at the corresponding cell range's location.
+
+Typical usages of heatmaps:
+
+- Display the magnitude of a data set over two dimensions;
+- In retail matrix, manufacturing diagram, and population maps;
+- For marketing goals and analytics, reflecting on user behavior on specific web pages;
+- ...
+
+To create a heatmap in Taipy, you must set the property [*type*](../../chart/#p-type)
+of the chart control to "heatmap".
+
+## Key properties
+
+| Name            | Value           | Notes   |
+| --------------- | ------------------------- | ------------------ |
+| [*type*](../chart.md#p-type)      | `heatmap`              |  |
+| [*x*](../chart.md#p-x)            | x values               |  |
+| [*y*](../chart.md#p-y)            | y values               |  |
+| [*z*](../chart.md#p-z)            | Main variable values.  |  |
+| [*options*](../chart.md#p-options)  | dictionary  | `colorscale` can be used to set a color scale.<br/>`showscale` can be set to False to hide the color scale.  |
```

Comment:
I thought showscale would be in layout ?

Reply:
Not for heatmaps...